screens/cart/Cart.css

```diff
@@ -0,0 +1,135 @@
+/* cart */
+.cart-container {
+  margin: 1rem;
+}
+
+.cart-wrapper {
+  max-width: 80%;
+  margin: auto;
+}
+
+.cart-title {
+  text-align: center;
+  margin: 1rem 0;
+}
+
+.cart-main {
+  display: grid;
+  grid-template-columns: 3fr 2fr;
+  gap: 1rem;
+}
+
+.cart-main > * {
+  box-shadow: var(--card-box-shadow);
+  border-radius: 5px;
+}
+
+/* products css */
+.cart-product {
+  max-width: 100%;
+  background-color: var(--card-box-shadow);
+  height: auto;
+}
+
+.card-horizontal {
+  display: flex;
+}
+
+.card-horizontal-img {
+  width: 40%;
+  padding: 10px;
```

Comment:
Can we use relative unit here as well? As it will help in being consistent and also in responsiveness.

Reply:
Yes, I'll keep this in mind, actually below 10px I though of using px.